INSTALL.md

```diff
@@ -577,6 +577,11 @@ as configuration option, you must ensure that it's valid for both the C and
 the C++ compiler.  If not, the C++ build test will most likely break.  As an
 alternative, you can use the language specific variables, `CFLAGS` and `CXXFLAGS`.
 
+### no-bulk
+
+Build only a minimal set of features.
+This is used for CI build tests of the project.
```

Comment:
One concern I have with this is that we might want to tweak what this option covers from time to time. By making it a config option like this we are "publishing" it for user use. One possibility is just to say that this is considered a developer option and should not be considered stable. The features disabled may change from one patch release to the next.

Reply:
That's why I did not enumerate the concrete features disabled this way.
Following your concern, I've re-phrased the documentation as follows, making more clear that this is something internal:
```
Build only some minimal set of features.
This is a developer option used internally for CI build tests of the project.

```